### PR TITLE
Different email workflow Completed

### PR DESCRIPTION
**Pull request by Lord-Chris**
https://user-images.githubusercontent.com/58702861/136202146-2bf3fe4d-863d-4d6a-8324-26bf39d8b36d.mp4

### CODE REVIEW

File: lib/ui/view/organization/select_email/select_email_view.dart [hunk 1]
```diff
@@ -14,109 +14,103 @@ class SelectEmail extends StatelessWidget {
...
-        builder: (context, model, child) => Scaffold(
-              appBar: AppBar(
-                title: Text(
-                  model.getScreenTitle(method),
-                  style: const TextStyle(
-                    fontSize: 16,
-                    color: AppColors.deepBlackColor,
+      viewModelBuilder: () => SelectEmailViewModel(),
+      builder: (context, model, child) => Scaffold(
+        appBar: AppBar(
```
Comment (jeremiahseun) [inline lib/ui/view/organization/select_email/select_email_view.dart:19]: Use ZuriAppBar

File: lib/ui/view/organization/select_email/select_email_view.dart [hunk 2]
```diff
@@ -14,109 +14,103 @@ class SelectEmail extends StatelessWidget {
...
+            iconSize: 12,
+            onPressed: () => model.pop(),
+            icon: const Icon(Icons.arrow_back_ios_outlined),
+            color: Colors.black,
+          ),
+        ),
+        body: Container(
+          margin: const EdgeInsets.fromLTRB(8.0, 20.0, 8.0, 0.0),
+          decoration: BoxDecoration(
+              color: Colors.white,
```
Comment (jeremiahseun) [inline lib/ui/view/organization/select_email/select_email_view.dart:38]: Use AppColors.whiteColor instead

**Review by jeremiahseun — CHANGES_REQUESTED** [review]
Please check through all files and make corrections

File: lib/ui/view/dm_user/dm_user_view.dart [hunk 3]
```diff
@@ -23,26 +22,36 @@ class DmUserView extends StatelessWidget {
...
+          body: Stack(
+            children: [
+              ExpandableTextFieldScreen(
+                hintText: 'Message ${model.receiver.username}',
+                sendMessage: (String message) {
+                  model.sendMessage();
+                  FocusScope.of(context).requestFocus(FocusNode());
+                  _scrollController
+                      .jumpTo(_scrollController.position.maxScrollExtent);
+                  // duration: Duration(milliseconds: 500),
```
Comment (jeremiahseun) [inline lib/ui/view/dm_user/dm_user_view.dart:51]: Please remove commented codes that are not used

File: lib/ui/view/organization/organization_url/organization_url_viewmodel.dart [hunk 4]
```diff
@@ -23,7 +23,6 @@ class OrganizationUrlViewModel extends BaseViewModel {
   final api = OrganizationApiService();
```
Comment (jeremiahseun) [inline lib/ui/view/organization/organization_url/organization_url_viewmodel.dart:23]: Hmm... Why not use locator?

File: lib/ui/view/organization/select_email/select_email_view.dart [hunk 5]
```diff
@@ -14,109 +15,93 @@ class SelectEmail extends StatelessWidget {
...
+        appBar: ZuriAppBar(
+          title: model.getScreenTitle(method),
+          whiteBackground: true,
+          leading: Icons.arrow_back_ios_outlined,
+          leadingPress: () => model.pop()
+        ),
+        body: Container(
+          margin: const EdgeInsets.fromLTRB(8.0, 20.0, 8.0, 0.0),
+          decoration: BoxDecoration(
+              color: Colors.white,
```
Comment (jeremiahseun) [inline lib/ui/view/organization/select_email/select_email_view.dart:29]: You have not changed the color yet. 
Use AppColors.whiteColor

Comment (Lord-Chris) [reply lib/ui/view/organization/select_email/select_email_view.dart:29]: Done

**Pull request closed without merging**